servicecontrol/license.md

```diff
@@ -29,12 +29,14 @@ To import a license using PowerShell:
 Import-ServiceControlLicense <LicenseFile>
 ```
 
-### Deprecated Licensing method
-It is also possible to apply a license to an individual instance rather than using the registry method. This can be done by by creating a `license` folder under the installation path of an instance and copying the `license.xml` to that directory.
-Adding a license in this manner is deprecated.   The ServiceControl Management Utility and PowerShell module only manage the registry license location
+### Instance Licensing
+
+In Version 1.17 and below, a license can be applied to an individual instance rather than using a license installed in the registry. To do this, copy the `license.xml` file to a `license` folder under the installation path of the instance.
```

Comment:
@mikeminutillo We've not taken this ability away in 1.18 or even in 1.19 yet. I read this as it no longer works for newer versions,